src/running-the-course.md

```diff
@@ -35,7 +32,9 @@ Before you run the course, you will want to:
    Using your laptop will also allow you to fix typos as you or the course
    participants spot them.
 
-1. Let people solve the exercises by themselves or in small groups. Make sure to
+1. Let people solve the exercises by themselves or in small groups.
+   We typically spend 30-45 minutes on exercises in the morning and in the afternoon.
+   Make sure to
    ask people if they're stuck or if there is anything you can help with. When
    you see that several people have the same problem, call it out to the class
    and offer a solution, e.g., by showing people where to find the relevant
```

Comment:
You might mention that reviewing the solutions is important. In a few cases there's even a morsel of additional course content in there! To my shame, I skipped this when I was instructing since I got so little response from students after the exercises.

Reply:
Yeah, good point! I added a tiny note about it.

I've found it hard to find time for this, unless the students a super quick to solve them. It's something we will need to try and improve in the future.